test/skipper/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://raw.githubusercontent.com/zalando/skipper/master/docs/kubernetes/deploy/deployment/rbac.yaml
+  - https://raw.githubusercontent.com/zalando/skipper/master/docs/kubernetes/deploy/deployment/deployment.yaml
+
+patches:
+  - target:
+      kind: Deployment
+      name: skipper-ingress
+    patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: skipper-ingress
+      spec:
+        template:
+          metadata:
+            annotations:
+              prometheus.io/path: /metrics
+              prometheus.io/port: "9911"
+              prometheus.io/scrape: "true"
+          spec:
+            nodeSelector:
+              $patch: delete
+            affinity:
+              $patch: delete
+            containers:
+              - name: skipper-ingress
+                image: registry.opensource.zalan.do/pathfinder/skipper:v0.11.135
```

Comment:
Should this be latest to figure out if skipper upstream broke things?

Reply:
According to Skipper docs there is no :latest provided!?

Reply:
Seems to work
```
% docker run  -it registry.opensource.zalan.do/pathfinder/skipper:latest skipper -version 
Skipper version v0.11.135 (commit: 1d764b3)
```

Reply:
I try to omit to tell users that they can use latest, but in this case it seems to be the better option.

Reply:
Thanks for this "secret" knowledge :smile: 

Adusted to latest with the next PR: https://github.com/o11n/flagger/pull/4/commits/40d41d2a0c07ea5d529f5bb7bd79227b36e0c041